pytorch_lightning/trainer/trainer.py

```diff
@@ -458,7 +460,7 @@ def __init__(
         # override dist backend when using tpus
         if self.on_tpu:
             self.init_tpu()
-            self.current_tpu_idx = None
+            self.current_tpu_idx = self.tpu_id
```

Comment:
do we need this extra attribute `current_tpu_idx`?

Reply:
I haven't noticed it being used anywhere. I assumed it can be used for logging during training.

Reply:
I do not think it is needed

Reply:
Have omitted it :)